allocate.go

```diff
@@ -377,6 +379,11 @@ func UserAgent(userAgent string) ExecAllocatorOption {
 	return Flag("user-agent", userAgent)
 }
 
+// RemoteDebuggingPort is the command line option to set remote-debugging-port.
+func RemoteDebuggingPort(port string) ExecAllocatorOption {
```

Comment:
Remove this API. Most users shouldn't be setting the parameter at all, and those who know what they're doing can just call Flag directly.

Reply:
ok